app/upgrades/v19_2/upgrades_test.go

```diff
@@ -82,9 +83,9 @@ func TestAddCodeToERC20Extensions(t *testing.T) {
 					require.Nil(t, acc)
 					return
 				}
-				ethAddr := common.BytesToAddress(acc.GetAddress().Bytes())
-				codeHash := network.App.EvmKeeper.GetCodeHash(ctx, ethAddr)
-				require.Equal(t, codeHash.String(), expCodeHash.String())
+				ethAcct, ok := acc.(*types.EthAccount)
+				require.True(t, ok)
+				require.Equal(t, ethAcct.CodeHash, expCodeHash.String())
```

Comment:
We should remove old migration since we decided to not keep them along the repo anymore.